malt/converters/slices.py

```diff
@@ -52,6 +52,23 @@ def visit_Assign(self, node):
     if replacement is not None:
       return replacement
     return node
+  
+  def visit_Slice(self, node):
+    node = self.generic_visit(node)
+    if not isinstance(node, (gast.Slice)):
+      return node
+    # This guard is required or the following example will produce a wrong
+    # data shape.
+    #   @qjit(autograph=True)
+    #   def expand_by_two(x):
+    #     print(*.x.shape[1:])
+    if node.lower is None or node.upper is None:
+      return node
+    template = """
+      slice(lower, upper)
```

Comment:
What is slice? Is that an existing operator in autograph?

Reply:
It's a Python builtin function.